.github/workflows/ci.yml

```diff
@@ -29,45 +29,43 @@ jobs:
     name: Simulate SW on Snitch Cluster w/ Verilator
     runs-on: ubuntu-22.04
     container:
-      image: ghcr.io/pulp-platform/snitch_cluster:main
+      image: ghcr.io/pulp-platform/snitch_cluster:test-ci
     steps:
       - uses: actions/checkout@v2
         with:
           submodules: 'recursive'
       - name: Build Software
         run: |
           bender vendor init
-          make -C target/snitch_cluster sw
+          make -C target/snitch_cluster sw -j
```

Comment:
Parallel make is very different from serial invocation and can cause _spectacular_ issues when the targets are not properly written (I speak from experience). I trust that you know this and tested this well.

Reply:
This goes for all following cases, too.

Reply:
I tested it just by invoking the targets, which the CI also does. But I will drop the relevant commit then; the speedup on the overall CI time should not be so significant anyways.